### ISSUE TITLE
problem with loss.py

### ISSUE THREAD

**Issue by EKELE-NNOROM** [philip-huang/PIXOR#15]
In loss.py shouldn't line 85 
loss = loss(pred, label)
be 
loss = loss.forward(pred, label)
